homeassistant/components/vizio/config_flow.py

```diff
@@ -50,6 +51,32 @@ def _config_flow_schema(input_dict: Dict[str, Any]) -> vol.Schema:
     )
 
 
+class VizioOptionsConfigFlow(config_entries.OptionsFlow):
```

Comment:
Why move this?

Reply:
I wasn't able to set the return type for `VizioConfigFlow.async_get_options_flow` without moving it. Do you think this is unnecessary?

Reply:
Then it's ok.